src/org/ods/services/GitService.groovy

```diff
@@ -124,12 +124,17 @@ class GitService {
         ).trim()
     }
 
-    /** Looks in commit message for string '[ci skip]', '[ciskip]', '[ci-skip]' and '[ci_skip]'. */
+    /** Looks in commit message for the following strings
+     *  '[ci skip]', '[ciskip]', '[ci-skip]', '[ci_skip]',
+     *  '[skip ci]', '[skipci]', '[skip-ci]', '[skip_ci]',
+     *  '***NO_CI***', '***NO CI***', '***NOCI***', '***NO-CI***'
+     */
     boolean isCiSkipInCommitMessage() {
-        return script.sh(
-            returnStdout: true, script: 'git show --pretty=%s%b -s',
+        def gitCommitMessage = script.sh(
+            returnStdout: true, script: 'git show --pretty=%s -s',
             label: 'check skip CI?'
-        ).toLowerCase().replaceAll('[\\s\\-\\_]', '').contains('[ciskip]')
+        ).toLowerCase().replaceAll('[\\s\\-\\_]', '')
+        return gitCommitMessage.contains('[ciskip]') || gitCommitMessage.contains('[skipci]') || gitCommitMessage.contains('***noci***')
```

Comment:
It would be nice to have this tested. Looking at `GitServiceSpec` there is no existing test right now, so totally understand you did not add one. But it would be great to have this. Want to add one?

Reply:
Sure, I'm happy to do it, but my groovy skills are based more on "copy&paste"-ish than real knowledge...
How do I call the function correctly, as I have only found the following reference for it and wondering if the missing prefix `is` intended?
https://github.com/opendevstack/ods-jenkins-shared-library/blob/4cdd980a979ffbf148b528647aa9dba94fa31307/src/org/ods/component/Pipeline.groovy#L325-L328:

Reply:
I was suggesting to simply add a test in `GitServiceSpec`. I think you do not need to go all the way to test integration with `Pipeline` (would be great of course but let's take small steps). If you look into `GitServiceSpec`, you should see an example that you can copy&paste ;)